docs/hotfix.md

```diff
@@ -0,0 +1,70 @@
+# How to make a hotfix release
+
+This document describes how to generate a hotfix release for backplane. 
+
+### When to make a hotfix release
+
+- A critical bug has been found
+- Security vulnerabilities have been encountered
+- Something broke in the last/latest release
+
+### Hotfix release process technical guidelines
+
+Please make sure to adhere to these following guidelines below for the hotfix release process : 
+
+#### 1. Create an OSD card
+
+- Create an OSD card with title : **Backplane Hotfix Release for <$reason>**
+- In the description of the card , mention details about the intention/reason behind this hotfix.
+- Add backplane-hotfix label to it.
+
+#### 2. Create pull request 
+
+The main aim is to put hotfixes to production without promoting all changes from the main branch.
```

Comment:
I do not think we have done this before, it would be easier to maintain if we just cut a new release in main branch for hotfix